test/test_jit.py

```diff
@@ -1201,7 +1201,7 @@ def test_arg_configurations(self):
 
         @torch.jit.compile(nderivs=0)
         def fn(*args):
-            in_vars, _ = torch._C._jit_flatten(args)
+            in_vars, _ = torch.jit._flatten(args)
```

Comment:
Looks like `ONNX exporter` doesn't have merge permission on this file. changing it can take longer

Reply:
I can change this one back.

Reply:
Done